src/equations/compressible_navier_stokes_1d.jl

```diff
@@ -81,7 +81,7 @@ or the entropy variables
 ```
 where
 ```math
-w_2 = \frac{\rho v1}{p},\, w_3 = -\frac{\rho}{p}
+w_2 = \frac{\rho v}{p},\, w_3 = -\frac{\rho}{p}
```

Comment:
```suggestion
w_2 = \frac{\rho v_1}{p},\, w_3 = -\frac{\rho}{p}
```

Reply:
I agree with @JoshuaLampert , for consistency with the Euler equations we should keep this as `v_1`